faq/tidb.md

```diff
@@ -930,14 +930,15 @@ The total read capacity has no limit. You can increase the read capacity by addi
 
 #### The error message `transaction too large` is displayed
 
-As distributed transactions need to conduct two-phase commit and the bottom layer performs Raft replication, if a transaction is very large, the commit process would be quite slow and the following Raft replication flow is thus struck. To avoid this problem, we limit the transaction size:
+As distributed transactions need to conduct two-phase commit and the underlying implementation performs Raft replication. If a transaction is very large, the commit process would be quite slow and it's more likely to conflict with other transactions. Morever, the rollback of transactions lead to unnecessary performance penalty. To avoid this problem, we limit the transaction size:
```

Comment:
```suggestion
Distributed transactions need to conduct two-phase commit and the underlying implementation performs Raft replication. If a transaction is very large, the commit process would be quite slow and this transaction is more likely to conflict with other transactions. Moreover, the rollback of a failed transaction leads to an unnecessary performance penalty. To avoid these problems, we limit the transaction size:
```